.gitignore

```diff
@@ -12,3 +12,8 @@ dfp_drivers
 aducm_project
 aducm_build
 
+*.a
+*.o
+*.d
+*.hex
+*.hdf
```

Comment:
I like seeing my .hdf's with a git status :D

Reply:
Also I would like to see here: *.a, .vscode
I am for *.hdf in .gitignore

Reply:
As per slack vote I will keep .hdf